pkgs/applications/editors/emacs/elisp-packages/manual-packages/wat-mode/default.nix

```diff
@@ -1,9 +1,8 @@
-# Manually packaged until it is upstreamed to melpa
-# See https://github.com/devonsparks/wat-mode/issues/1
```

Comment:
Not sure if we should delete this.  If you really want to do this, please do so in a separate commit.

Reply:
https://github.com/NixOS/nixpkgs/pull/329232